pom.xml

```diff
@@ -404,7 +406,7 @@
         <version.org.jboss.mod_cluster>1.4.1.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>5.10.6.Final</version.org.jboss.narayana>
         <version.org.jboss.openjdk-orb>8.1.4.Final</version.org.jboss.openjdk-orb>
-        <version.org.jboss.resteasy>3.14.0.Final</version.org.jboss.resteasy>
+        <version.org.jboss.resteasy>3.15.0.Alpha1</version.org.jboss.resteasy>
```

Comment:
@asoldano @ronsigal Is there a 3.15.0.Final yet?

Reply:
Hey @kabir, that's exactly what I'm working on. As soon as I (or someone - Jason Lee offered to look at it) can figure out that OpenTracing test failure, I'm ready to go.